src/config.js

```diff
@@ -42,26 +42,29 @@ module.exports = (store) => {
      * @returns {void}
      */
     set (key, value) {
```

Comment:
this is probably contentious, because it would be breaking, but I feel as though explicitly setting a null key would be better than hot swapping the key/value, but this would be a breaking change. So I'll keep it going.